packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -528,9 +514,10 @@ export const GiantTrackTile = ({
       includeFavorite: false,
       includeTrackPage: false,
       isArtistPick,
-      includeEmbed: !(isUnlisted || isStreamGated),
```

Comment:
we want this change because it's your own private track or link was shared with you and if you see the menu you already have access, right?

Reply:
ah just checked and it results in an error, so ill update the track-tile one to not include it, great catch!